### PR TITLE
Web scraper logging

### PR DESCRIPTION
**Pull request by Musa-Mabasa**
This branch adds logs to the console using `Winston` to format the logs.

### CODE REVIEW

File: src/delete.ts [hunk 1]
```diff
@@ -1,92 +1,110 @@
...
+      tls: {
+        rejectUnauthorized: false,
+      },
+    }
+  )
+    .then((resp) => {
+      logger.info(resp.headers);
+      resp.json();
+    })
+    .then((res) => console.log(res))
```
Comment (tanadendere) [inline src/delete.ts]: You forgot a` console.log `here

**Review by tanadendere — COMMENTED** [review]
looks good, just need to remove the one `console.log ` but otherwise, well done on using Winston!

**Review by LesediMoko — APPROVED** [review]
Besides the one console log that Tana mentioned, everything looks good! 👍

File: src/delete.ts (hunk 1 above)
Comment (Musa-Mabasa) [reply src/delete.ts]: Thank you, fixed 👍

**Review by tanadendere — APPROVED** [review]

**Pull request merged**